Readme.md

```diff
@@ -4,3 +4,9 @@ Available here: https://drive.google.com/drive/folders/1l5AL2O7nnpnsh73UjqjLYcBo
 # Comment
 I've pulled data from static bahis and had some fun formatting it (all the different forms submissions are in one table with the actual data stored as key:value pairs). I have uploaded the data for you into google drive, those are files in a format `formdata_*.csv`.
 You will notice however that it is not possible to find out what different indexes mean in many cases. I leave that for you to find out as you go through whichever dataset you find most interesting. The tables that contain keys for submitted data are in `input/STATICBAHIS*.csv` files, though at this point I'm not sure what is what.
+
+#  Setting up the environment for data pileline
```

Comment:
Markdown files should only ever have one "top-level" header (i.e. a single `#`) - we should probably add one at the top that says (for example) `# BAHIS Data Processing Pipelines` and make `# Comment` a `##` as well as this header.